docs/devupdate.md

```diff
@@ -62,6 +62,61 @@ shortname,fullname,category,summary,cohort,student,cohort,teacher
 
 </details>
 
+## Activity icons
+
+<Since version="4.4" issueNumber="MDL-78284" />
+
+### New designs
+
+The activity icons have been updated to prioritize readability and transform the existing filled square shapes with white icons into outlined colored icons against a transparent background. This modification ensures that the icons are not only more accessible but also easily distinguishable for users.
+When upgrading the SVG files, the .png files have been removed.
+
+:::tip Considerations for creating/updating SVG icons
+
+- Please make sure that the icons are filled with #212529 and that the background is transparent.
```

Comment:
Using monospace  for `#212529` would make it look better.

Reply:
@kabalin, thanks a lot for the suggestion! I've just updated the patch! <3